From: Mel Gorman <mgorman@suse.de>
Date: Wed, 18 May 2011 14:01:11 +0100
Subject: [PATCH] thp: CONFIG_TRANSPARENT_HUGEPAGE
References: THP core (fate #311931)
Git-commit: 4c76d9d1fb9b21fa10c9e4c1fab2875018a88aa1
Patch-mainline: v2.6.38-rc1

Add config option.

Signed-off-by: Andrea Arcangeli <aarcange@redhat.com>
Acked-by: Rik van Riel <riel@redhat.com>
Acked-by: Mel Gorman <mgorman@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Mel Gorman <mgorman@suse.de>
Automatically created from "patches.suse/mm-thp-config.patch" by xen-port-patches.py

 mm/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- sle11sp2-2011-05-31.orig/mm/Kconfig	2011-05-31 10:04:36.000000000 +0200
+++ sle11sp2-2011-05-31/mm/Kconfig	2011-05-31 11:40:05.000000000 +0200
@@ -191,7 +191,7 @@ config MIGRATION
 
 config TRANSPARENT_HUGEPAGE
 	bool "Transparent Hugepage Support"
-	depends on X86 && MMU
+	depends on X86 && !XEN && MMU
 	select COMPACTION
 	help
 	  Transparent Hugepages allows the kernel to use huge pages and

